web/src/app/profile/ProfileComponent.js

```diff
@@ -1,40 +1,68 @@
-import { Col, Grid, Image, Nav, NavItem, Row } from 'react-bootstrap';
+import { Col, Grid, Image, Nav, NavItem, Row, PageHeader } from 'react-bootstrap';
 import React, { Component } from 'react';
+import AlertComponent from './AlertComponent/AlertComponent';
 import PropTypes from 'prop-types';
+import styles from './ProfileStyles.scss'
+
+const ACCOUNT_TAB = 'account';
+const OPEN_CONTRACTS_TAB = 'open_contracts';
+const COMPLETED_CONTRACTS_TAB = 'completed_contracts';
+const ALERT_TAB = 'alerts';
 
 class ProfileComponent extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      activeTab: ACCOUNT_TAB
+    };
+  }
+
+  onTabClick(newActiveTab) {
+    this.setState({
+      activeTab: newActiveTab
+    });
+  }
+
   getContent() {
-    if (this.props.activeTab === 'account') {
+    if (this.state.activeTab === ACCOUNT_TAB) {
       return this.getAccountContent();
+    } else if (this.state.activeTab === OPEN_CONTRACTS_TAB) {
+      return this.getOpenContractsContent();
+    } else if (this.state.activeTab === COMPLETED_CONTRACTS_TAB) {
+      return this.getCompletedContractsContent();
+    } else {
```

Comment:
I dont like 'defaulting' to the alert tab. Id rather this be a else if (this.state.activeTab === ALERT_TAB) check and the else block would contain a console.error('invalid tab selected: ' +this.state.activeTab )

Reply:
Hmm...I kinda disagree. Since this is UI state code I think it is better practice to default to something. We should always show something on the UI. If we add an else with nothing and then we mismanage the state in another part of the code, we do not want the user to be confused by showing them a blank screen. We should default to something.